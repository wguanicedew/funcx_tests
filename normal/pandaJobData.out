{43451671: {StatusCode: 0, PandaID: 43451671, prodSourceLabel: test, swRelease: NULL, homepackage: NULL, transformation: http://pandaserver-doma.cern.ch:25080/trf/user/runGen-00-00-02, jobName: user.wguan.117eea76-ba54-42fa-8fd6-7ed6d1a2b705/.43451671, jobDefinitionID: 0, cloud: Rubin, inFiles: , dispatchDblock: , dispatchDBlockToken: , dispatchDBlockTokenForOut: NULL,NULL, outFiles: user.wguan.117eea76-ba54-42fa-8fd6-7ed6d1a2b705.log.151988.000003.log.tgz,user.wguan.151988._000003.myout.txt, destinationDblock: user.wguan.117eea76-ba54-42fa-8fd6-7ed6d1a2b705.log/,user.wguan.117eea76-ba54-42fa-8fd6-7ed6d1a2b705_myout.txt/, destinationDBlockToken: NULL,NULL, prodDBlocks: , prodDBlockToken: , realDatasets: user.wguan.117eea76-ba54-42fa-8fd6-7ed6d1a2b705.log/,user.wguan.117eea76-ba54-42fa-8fd6-7ed6d1a2b705_myout.txt/, realDatasetsIn: , fileDestinationSE: NULL,NULL, logFile: user.wguan.117eea76-ba54-42fa-8fd6-7ed6d1a2b705.log.151988.000003.log.tgz, logGUID: d17c54f5-32d5-4810-8b56-f5cd2b46122b, jobPars: -j \\ --sourceURL https://pandaserver-doma.cern.ch:25443 -r . -l  -o \{'myout.txt': 'user.wguan.151988._000003.myout.txt'}\ -p \python%20test_parsl_funcx.py%20%3E%20myout.txt\, attemptNr: 1, GUID: , checksum: , fsize: , scopeIn: , scopeOut: user.wguan, scopeLog: user.wguan, ddmEndPointIn: , ddmEndPointOut: ,, destinationSE: NULL, prodUserID: Wen Guan, maxCpuCount: 0, minRamCount: 1800, maxDiskCount: 300, cmtConfig: NULL, processingType: panda-client-1.5.22-jedi-run, transferType: NULL, sourceSite: NULL, currentPriority: 1000, taskID: 151988, coreCount: 1, jobsetID: 151988, reqID: 151988, nucleus: NULL, maxWalltime: 0, ioIntensity: null, ioIntensityUnit: null, nSent: 3, inFilePaths: }}
