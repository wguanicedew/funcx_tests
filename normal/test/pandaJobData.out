{43451671: {StatusCode: 0, PandaID: 43451671, prodSourceLabel: test, swRelease: NULL, homepackage: NULL, transformation: http://pandaserver-doma.cern.ch:25080/trf/user/runGen-00-00-02, jobName: user.wguan.117eea76-ba54-42fa-8fd6-7ed6d1a2b705/.43451671, jobDefinitionID: 0, cloud: Rubin, inFiles: , dispatchDblock: , dispatchDBlockToken: , dispatchDBlockTokenForOut: NULL,NULL, outFiles: user.wguan.117eea76-ba54-42fa-8fd6-7ed6d1a2b705.log.151988.000003.log.tgz,user.wguan.151988._000003.myout.txt, destinationDblock: user.wguan.117eea76-ba54-42fa-8fd6-7ed6d1a2b705.log/,user.wguan.117eea76-ba54-42fa-8fd6-7ed6d1a2b705_myout.txt/, destinationDBlockToken: NULL,NULL, prodDBlocks: , prodDBlockToken: , realDatasets: user.wguan.117eea76-ba54-42fa-8fd6-7ed6d1a2b705.log/,user.wguan.117eea76-ba54-42fa-8fd6-7ed6d1a2b705_myout.txt/, realDatasetsIn: , fileDestinationSE: NULL,NULL, logFile: user.wguan.117eea76-ba54-42fa-8fd6-7ed6d1a2b705.log.151988.000003.log.tgz, logGUID: d17c54f5-32d5-4810-8b56-f5cd2b46122b, jobPars: -j \\ --sourceURL https://pandaserver-doma.cern.ch:25443 -r . -l  -o \{'myout.txt': 'user.wguan.151988._000003.myout.txt'}\ -p \python%20test_parsl_funcx.py%20%3E%20myout.txt\, attemptNr: 1, GUID: , checksum: , fsize: , scopeIn: , scopeOut: user.wguan, scopeLog: user.wguan, ddmEndPointIn: , ddmEndPointOut: ,, destinationSE: NULL, prodUserID: Wen Guan, maxCpuCount: 0, minRamCount: 1800, maxDiskCount: 300, cmtConfig: NULL, processingType: panda-client-1.5.22-jedi-run, transferType: NULL, sourceSite: NULL, currentPriority: 1000, taskID: 151988, coreCount: 1, jobsetID: 151988, reqID: 151988, nucleus: NULL, maxWalltime: 0, ioIntensity: null, ioIntensityUnit: null, nSent: 3, inFilePaths: }}


"echo {jobData} > pandaJobData.out; export PANDA_JSID=harvester-{harvesterID}; export HARVESTER_ID={harvesterID}; export HARVESTER_WORKER_ID={workerID}; export OIDC_AUTH_DIR=$(pwd); export OIDC_AUTH_TOKEN=oidc_token; echo {token} > $OIDC_AUTH_DIR/$OIDC_AUTH_TOKEN; export OIDC_AUTH_ORIGIN={tokenOrigin}; export VO_ATLAS_SW_DIR=/tmp/; wget http://pandaserver-doma.cern.ch:25080/cache/schedconfig/{computingSite}.all.json -O queuedata.json; wget https://raw.githubusercontent.com/PanDAWMS/pilot-wrapper/master/runpilot2-wrapper.sh; chmod +x runpilot2-wrapper.sh; bash runpilot2-wrapper.sh --localpy --piloturl http://cern.ch/atlas-panda-pilot/pilot3-3.6.0.48.tar.gz  -t -s {computingSite} -r {computingSite} -q {pandaQueueName} -j {prodSourceLabel} -i {pilotType} -d --harvester-submit-mode PUSH -w generic --pilot-user ATLAS --queuedata-url http://pandaserver-doma.cern.ch:25080/cache/schedconfig/{computingSite}.all.json --storagedata-url https://datalake-cric.cern.ch/api/atlas/ddmendpoint/query/?json --noproxyverification"
